json2db.pl

```diff
@@ -14,10 +14,15 @@
 
 foreach my $pkg (@$pkgs) {
     #print $pkg->{package}," ", $pkg->{status},"\n";
-    $pkg->{status} =~ s/unreproducible/FTBR/;
+    my $status = $pkg->{status};
+    $status =~ s/unreproducible/FTBR/;
+    $status =~ s/BAD/FTBR/;
+    $status =~ s/UNKWN/FTBFS/;
+    $status =~ s/GOOD/reproducible/;
```

Comment:
Overall it looks fine:

- `unreproducible` does not exist as status in rebuilderd
- `BAD` can mean either `FTBR` or `FTBFS`, the difference is currently not tracked (everything that isn't `GOOD`)
- `UNKWN` is the "pending" status, waiting to be picked up by a worker

Reply:
> For UNKWN - would it be possible to get an export with the last known state?

With the current implementation the previous status is not available unfortunately. If a package version disappears from the package database (core.db, extra.db) it also gets removed from the rebuilderd database.